runtime/module.go

```diff
@@ -81,15 +81,12 @@ type AppInputs struct {
 	AppConfig      *appv1alpha1.Config
 	Config         *runtimev1alpha1.Module
 	AppBuilder     *AppBuilder
-	Modules        map[string]AppModuleWrapper
```

Comment:
We can remove `AppModuleWrapper` in runtime/wrappers.go, right?

Reply:
TBH I would even remove `WrapAppModuleBasic` and all `ProvideModuleBasic`s. Does depinject need the AppModuleBasics? Maybe only provide the appmodule.AppModules?

Reply:
I removed `AppModuleWrapper`. I'll look into removing `WrapAppModuleBasic`. It's needed for codec instantiation, but maybe I can find a workaround.

Reply:
I removed WrapModuleBasic. A bunch of boilerplate gone now. Should I go ahead and make the rest of AppModuleBasic extension interfaces? In particular `HasInterfaces` and `HasAmino`?

Reply:
Yes, I think that's a good idea. Totally fine to do in a follow-up too, as this already has approvals. It's not breaking anyways, right?